### PR TITLE
Attempting to integrate Google Analytics

### PR DESCRIPTION
**Pull request by ravindk89**
This is testing to see if I can get Google Analytics to work properly. This does appear to load the script on every page in the docs. It's a little ugly, but its working. I *think* it will only *actually* work once I put it on the webserver though, so before going that route I want to gather thoughts.

### CODE REVIEW

**Comment by ravindk89** [comment]
Added crazyegg script @rushenn @kaankabalak 

I don't see errors in my local using python http.server, so I *think* this should work once on the actual docs server.

File: source/_templates/layout.html [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
 {% extends "!layout.html" %}
 
+
 {# Custom CSS overrides #}
 
+{% block extrahead %}
+
+  {{ super() }}
+
+   <script type="text/javascript" src="//script.crazyegg.com/pages/scripts/0088/9671.js" async="async"></script>
```
Comment (rushenn) [inline source/_templates/layout.html:19]: It appears the GA scripts are placed inside the `body` tag. Can we place them inside the `head`?

Comment (ravindk89) [reply source/_templates/layout.html:19]: Done

**Review by rushenn — APPROVED** [review]
LGTM

**Pull request merged**